0xxx-scaling-mobile-mappers.md

```diff
@@ -0,0 +1,60 @@
+- **Random Skyhook check:** Mappers must grant the app Wi-Fi permission. The app will perform periodic, random Skyhook checks to see if the Skyhook location aligns with the phone’s GPS. If there’s a mismatch in the last 7 days, the mapper won’t be automatically upgraded to a verification mapper. 
+- **Location trust of Hotspots being verified:** Hotspots with high location trust scores, verified by mappers or tower correlations, will serve as anchors to verify the mapper's location. If a mapper connects to a high-trust Hotspot but reports a different location, they’ll be banned from an automatic upgrade to verification mapper until the next 30-day cycle, after burning another $5.
+
+To enable mappers to perform their function, all mappers get access to all Hotspots on the Helium Mobile network, limited to a low per session data cap, like 5Mb per session or rate limited to a low data rate that is useful for mapping but not much else. The data used by a mapper is **not** rewardable to the Hotspot it transits. But mappers will receive a share of the mapping reward emissions.
+
+## Drawbacks
+Currently, the only way to become a mapper is by subscribing to a service provider. Removing this requirement might make the role of becoming a voted service provider on the network less appealing. However, most service providers are unable to handle or receive crypto due to regulatory uncertainty in many regions, so this issue seems less relevant in the short to medium term.
+
+By eliminating the need for mappers to be service provider customers, GPS confirmation through tower correlations can no longer be the primary security method for validating mappers. At the same time, the reliability of the proposed alternative random tests in this HIP is not fully proven. It’s likely that new methods for gaming the system may emerge, requiring future mitigation.
+
+Additionally, it’s unclear whether many users will be willing to burn $5 of MOBILE tokens each month just for the right to map. After launch, this requirement might need to be adjusted. Potential solutions could include modifying the fee, changing the frequency of payments, or offering some amount of usable data on the MOBILE network for mappers.
+
+## Rationale and Alternatives
+One alternative would be for an existing network service provider, like Helium Mobile, to introduce a $5 paid plan that includes a small amount of data on the Helium Mobile network alongside mapping privileges. However, this approach doesn’t allow third-party entities, such as crypto wallet providers or app developers, to contribute to the proliferation of mappers, which if they could, would lead to a more centralized network.
```

Comment:
```suggestion
One alternative would be for an existing network service provider, like Helium Mobile, to introduce a $5 paid plan that includes a small amount of data on the Helium Mobile network alongside mapping privileges. However, this approach doesn’t allow third-party entities, such as crypto wallet providers or app developers, to contribute to the proliferation of mappers, which, if they could, would lead to a more centralized network.
```